src/UKHO.SAP.MockAPIService/Services/z_adds_mat_info.cs

```diff
@@ -21,17 +22,22 @@ public z_adds_mat_info(IAzureBlobEventWriter azureBlobEventWriter, MockService m
         }
 
         [return: XmlElement("Z_ADDS_MAT_INFOResponse", Namespace = "urn:sap-com:document:sap:rfc:functions")]
-        public Z_ADDS_MAT_INFOResponse Z_ADDS_MAT_INFO([XmlElement("Z_ADDS_MAT_INFO", Namespace = "urn:sap-com:document:sap:rfc:functions")] Z_ADDS_MAT_INFO z_ADDS_MAT_INFO)
+        public Z_ADDS_MAT_INFOResponse Z_ADDS_MAT_INFO([XmlElement("IM_MATINFO", Namespace = "")] IM_MATINFO iM_MATINFO)
         {
+            Z_ADDS_MAT_INFO z_ADDS_MAT_INFO = new()
+            {
+                IM_MATINFO = iM_MATINFO
+            };
+
             string requestXML = ObjectXMLSerializer<Z_ADDS_MAT_INFO>.SerializeObject(z_ADDS_MAT_INFO).WriteXmlClosingTags();
 
-            Task.Run(async () => await _azureBlobEventWriter.UploadEvent(requestXML, z_ADDS_MAT_INFO.IM_MATINFO.CORRID, z_ADDS_MAT_INFO.IM_MATINFO.CORRID + '.' + RequestFormat));
+            Task.Run(async () => await _azureBlobEventWriter.UploadEvent(requestXML, iM_MATINFO.CORRID, iM_MATINFO.CORRID + '.' + RequestFormat));
 
             _mockService.CleanUp();
 
             return new Z_ADDS_MAT_INFOResponse()
             {
-                EX_MESSAGE = "Record successfully received for " + z_ADDS_MAT_INFO.IM_MATINFO.CORRID,
+                EX_MESSAGE = "Record successfully received for " + iM_MATINFO.CORRID,
```

Comment:
can we write message as "Record successfully received for CorrelationId :

Reply:
Updated